python/llm/src/ipex_llm/transformers/npu_models/convert_mp.py

```diff
@@ -48,6 +48,13 @@ def optimize_llm_pre(model: torch.nn.Module, qtype, mixed_precision,
 
     cpu_lm_head = os.environ.get("IPEX_LLM_CPU_LM_HEAD", "0") != "0"
 
+    # workaround for long input performance of llama3.2-3b and glm-edge-4b CW
+    if os.environ.get("IPEX_LLM_NPU_DISABLE_COMPILE_OPT") is None:
```

Comment:
do we need this line here ?

Reply:
> do we need this line here ?

I feel we'd better not change user setting, so this internal setting works when not specified : )